nflow-engine/src/main/resources/scripts/db/db2.create.ddl.sql

```diff
@@ -0,0 +1,146 @@
+-- production tables
+
+create table nflow_workflow (
+  id int primary key generated always as identity,
+  status varchar(32) not null check (status in ('created', 'executing', 'inProgress', 'finished', 'manual')),
+  type varchar(64) not null,
+  root_workflow_id integer default null,
+  parent_workflow_id integer default null,
+  parent_action_id integer default null,
+  business_key varchar(64),
+  external_id varchar(64) not null,
+  state varchar(64) not null,
+  state_text varchar(128),
+  next_activation timestamp(3),
+  external_next_activation timestamp(3),
+  executor_id int,
+  retries int not null default 0,
+  created timestamp(3) not null default current_timestamp,
+  modified timestamp(3) not null default current_timestamp,
+  executor_group varchar(64) not null,
+  workflow_signal int,
+  constraint nflow_workflow_uniq unique (type, external_id, executor_group)
+);
+
+-- create or replace function update_modified() returns trigger language plpgsql as '
```

Comment:
This commented stuff is Oracle related? Could be removed?

Reply:
It was just missing :). Integration tests didn't cover checking that modified-column auto-updates. Now they include at least one check.